src/linux/backend-v4l2.cpp

```diff
@@ -565,7 +565,7 @@ namespace librealsense
                 static const std::regex video_dev_pattern("(\\/video\\d+)$");
 
                 std::string path = "/sys/class/video4linux/" + name;
-                std::string real_path{};
+                std::string real_path {""};
```

Comment:
`std::string real_path` is not enough here?
What is the difference between
`std::string real_path`
and
`std::string real_path {""};`

Reply:
`std::string real_path;` and `std::string real_path = {""};` are the same

Reply:
If I started to revert the commit I can change `std::string real_path;` if it need.
But I don't know what KW will say us about this change.